ci/jenkins/Build.groovy.j2

```diff
@@ -93,9 +93,9 @@ stage('Build') {
     {{ m.upload_artifacts(tag='gpu', filenames=tvm_multilib, folders=microtvm_template_projects) }}
 
     // compiler test
-    sh "${docker_run} --no-gpu ${ci_gpu} ./tests/scripts/task_config_build_gpu_other.sh build2"
-    make("${ci_gpu} --no-gpu", 'build2', '-j2')
-    {{ m.upload_artifacts(tag='gpu2', filenames=tvm_multilib) }}
+    sh "${docker_run} --no-gpu ${ci_gpu} ./tests/scripts/task_clean.sh build",
```

Comment:
I think the very unhelpful Jenkins error [here](https://ci.tlcpack.ai/blue/organizations/jenkins/tvm/detail/fix_gpu_other_build/1/pipeline/#step-377-log-77) is actually complaining about this comma
```suggestion
    sh "${docker_run} --no-gpu ${ci_gpu} ./tests/scripts/task_clean.sh build"
```